pallet/account-migration/src/lib.rs

```diff
@@ -435,34 +418,6 @@ pub mod pallet {
 					);
 				}
 			}
-			if let Some(mut i) = <Identities<T>>::take(from) {
-				i.deposit = 0;
-				i.judgements.iter_mut().for_each(|(_, j)| {
-					if let Judgement::FeePaid(f) = j {
-						*f = 0;
-					};
-				});
-
-				migration::put_storage_value(
-					b"Identity",
-					b"IdentityOf",
-					&Twox64Concat::hash(&to.encode()),
-					i,
-				);
-			}
-			{
-				let mut rs = <pallet_identity::Pallet<T>>::registrars();
-
-				for r in rs.iter_mut().flatten() {
-					if r.account.0 == <AccountId32 as AsRef<[u8; 32]>>::as_ref(from)[..20] {
-						r.account = *to;
-
-						break;
-					}
-				}
-
-				migration::put_storage_value(b"Identity", b"Registrars", &[], rs);
-			}
```

Comment:
How do you deal with this identity in this pallet and the pallet-identity? Keep it for a while and then migrate to another solution?

Do you remember the way we deal with the lock balance about the identity in the account migration? We may need to take that into account.

Reply:
It doesn't use lock. It uses reserve. But we already free all reserve. So, delete it directly.

Reply:
Okay.